lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -625,7 +625,7 @@ defmodule ExUnitTest do
       assert output =~ "3 tests, 2 failures"
     end
 
-    test ":max_failures on setup all errors" do
+    test ":max_failures on setup_all errors" do
```

Comment:
`setup_all/1,2`?

Reply:
I think it is fine as is, it is not mean to be a complete reference and `setup_all` is enough to indicate the test purpose.